### PR TITLE
Cron: adding note for improvement

### PR DESCRIPTION
**Pull request by Ritikaa96**
Signed-off-by: Ritikaa96 <ritika@india.nec.com>

**Description**
Adding Note for valid start and end inputs in Cron scaler as mentioned in this comment [here](https://github.com/kedacore/keda/pull/2032#issuecomment-898603020)
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #[2031](https://github.com/kedacore/keda/issues/2031)

### CODE REVIEW

File: content/docs/1.5/scalers/cron.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ triggers:
...
 - `start` is the cron expression indicating the start of the cron schedule. Required.
 - `end` is the cron expression indicating the end of the cron schedule. Required.
+
+> **Notice:**
+> - **start and end should not be same.** For example:
+> ```yaml
+> start: 30 * * * *
+> end: 30 * * * *  # Invalid input
+>```
+> - The above schedule is invalid.
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/cron.md:37]: What about this?

```suggestion
> **Notice:**
> **Start and end should not be same.**
> 
> For example, the following schedule is not valid:
> ```yaml
> start: 30 * * * *
> end: 30 * * * *
>```
```

**Review by tomkerkhove — COMMENTED** [review]
Just a suggestion

File: content/docs/1.5/scalers/cron.md (hunk 1 above)
Comment (Ritikaa96) [reply content/docs/1.5/scalers/cron.md:37]: Sure @tomkerkhove I'll make the change. Any other suggestion?

**Comment by Ritikaa96** [comment]
@tomkerkhove Changed the note as suggested. PTAL and let me know if any other change is needed.

File: content/docs/1.5/scalers/cron.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ triggers:
...
+> **Notice:**
+> **Start and end should not be same.**
+>
+> For example, the following schedule is not valid:
+> ```yaml
+> start: 30 * * * *
+> end: 30 * * * *
+>```
+
 - `desiredReplicas` is the number of replicas to which the resource has to be scaled between the start and end of the cron schedule. Required.
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/cron.md:40]: I've missed this in the initial review but let's list all parameters first and then drop the notice.

```suggestion
- `end` is the cron expression indicating the end of the cron schedule. Required.
- `desiredReplicas` is the number of replicas to which the resource has to be scaled between the start and end of the cron schedule. Required.

> **☝ Notice - Start and end should not be same.**
>
> For example, the following schedule is not valid:
> ```yaml
> start: 30 * * * *
> end: 30 * * * *
>```

```

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/1.5/scalers/cron.md (hunk 2 above)
Comment (Ritikaa96) [reply content/docs/1.5/scalers/cron.md:40]: @tomkerkhove  sure thing.

**Comment by Ritikaa96** [comment]
@tomkerkhove shifted the note after  all parameters. PTAL.

**Comment by tomkerkhove** [comment]
Thank you!

**Pull request merged**